src/main/java/com/rcv/Tabulator.java

```diff
@@ -688,6 +688,22 @@ private OvervoteDecision getOvervoteDecision(Set<String> candidateIDSet) {
 
     // CVR indexes over the cast vote records to count votes for continuing candidateIDs
     for (CastVoteRecord cvr : castVoteRecords) {
+      // see if this cvr recipient is a continuing candidate
+      // in this case we can skip logic to determine who the cvr will count for
+      if (!cvr.isExhausted() &&
+          !(cvr.getCurrentRecipientOfVote() == null) &&
+          isCandidateContinuing(cvr.getCurrentRecipientOfVote())) {
+
+        // this vote stays with it's current recipient
+        incrementTallies(roundTally,
+            cvr.getFractionalTransferValue(),
+            cvr.getCurrentRecipientOfVote(),
+            roundTallyByPrecinct,
+            cvr.getPrecinct());
+        continue;
```

Comment:
Might be worth adding a comment here explaining why we don't call `logRoundOutcome`.